manifests/server/config.pp

```diff
@@ -39,6 +39,13 @@
     mode   => '0700',
   }
 
+  file { $ldap::server::server_run_directory:
+    ensure => directory,
+    owner  => $ldap::server::ldapowner,
+    group  => $ldap::server::ldapgroup,
+    mode   => '0700',
```

Comment:
This mode now for me on OpenBSD leads to the following messages every time puppet runs:

Notice: /Stage[main]/Ldap::Server::Config/File[/var/run/openldap]/mode: mode changed '0755' to '0700'
Info: Class[Ldap::Server::Config]: Scheduling refresh of Class[Ldap::Server::Service]

So any objection to change this to '0755' or otherwise make it OS dependent in params.pp?

Sebastian